source/_dashboards/tile.markdown

```diff
@@ -71,3 +75,102 @@ Alternatively, the card can be configured using YAML:
 ## Available color tokens
 
 Some color tokens are available to colorize the tile card : `primary`, `accent`, `disabled`, `red`, `pink`, `purple`, `deep-purple`, `indigo`, `blue`, `light-blue`, `cyan`, `teal`, `green`, `light-green`, `lime`, `yellow`, `amber`, `orange`, `deep-orange`, `brown`, `grey`, `blue-grey`, `black` and `white`.
+
+## Tile extras
```

Comment:
We should probably have at least 1 full-example  of card configuration that includes the `extras` key, maybe use `vacuum-commands` as the example as that has more than just `type`